upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -241,7 +241,9 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 
 	if b.cluster.Spec.Networking.Weave != nil {
 		key := "networking.weave"
-		version := "1.9.4"
+
+		// 1.9.5-kops.1 = 1.9.4 with WEAVE_MTU configured
+		version := "1.9.5-kops.1"
 
```

Comment:
Per http://semver.org/#spec-item-9 things with labels after the version are considered "pre-release" and have lower precedence than the final version number. So 1.9.4-kops.1 would be treated like a pre-release version of 1.9.4, and 1.9.4 would take precedence.